apel/parsers/sge.py

```diff
@@ -89,9 +89,9 @@ def _load_multipliers(self):
         hosts = xml_str.getElementsByTagName("host")
 
         for host in hosts:
-            host_name = dict(host.attributes.items())["name"]
+            host_name = dict(list(host.attributes.items()))["name"]
             for resource in host.getElementsByTagName("resourcevalue"):
-                resource_name = dict(resource.attributes.items())["name"]
+                resource_name = dict(list(resource.attributes.items()))["name"]
```

Comment:
Since the list is immediately consumed by dict, there is no advantage to the list. Just remove it.

That said, it seems superfluous to construct an entire `dict` only to access one item *by key*. You should be able to do something like `host.attributes["name"]` (or similar - Python 2 may need an explicitly named method).